src/pulse3D/plate_recording.py

```diff
@@ -591,7 +595,7 @@ def to_dataframe(self) -> pd.DataFrame:
             data[well_name] = pd.Series(interp_force_newtons_normalized)
 
             # add stim data
-            if output_stim_data:
+            if "aggregate_stim_timepoints_us_for_plotting" in locals():
```

Comment:
This probably isn't a good idea, we should just set `aggregate_stim_timepoints_us_for_plotting = None` right before line 567